apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/block-editor-nux.js

```diff
@@ -26,19 +27,18 @@ function WelcomeTour() {
 	const { show, isLoaded, variant, isManuallyOpened, isNewPageLayoutModalOpen } = useSelect(
 		( select ) => {
 			const welcomeGuideStoreSelect = select( 'automattic/wpcom-welcome-guide' );
-			const starterPageLayoutsStoreSelect = select( 'automattic/starter-page-layouts' );
 			return {
 				show: welcomeGuideStoreSelect.isWelcomeGuideShown(),
 				isLoaded: welcomeGuideStoreSelect.isWelcomeGuideStatusLoaded(),
 				variant: welcomeGuideStoreSelect.getWelcomeGuideVariant(),
 				isManuallyOpened: welcomeGuideStoreSelect.isWelcomeGuideManuallyOpened(),
-				isNewPageLayoutModalOpen: starterPageLayoutsStoreSelect?.isOpen(), // Handle the case where SPT is not initalized.
```

Comment:
This is the main thing we need to test. The purpose of the SPT store is to track whether the "new page template" screen is open or closed. Since that feature isn't always loaded, it used to be impossible for the store to be defined.

However, now that we directly import the store from the file, it seems more likely that the store stuff would at least be defined.

But on top of that, I'm not sure how it works when the store is imported in multiple different plugins that are technically part of different apps/scripts. ETK isn't one unified application -- instead, it's multiple smaller features which are ultimately individual scripts that get registered in PHP based on whether that feature should be loaded on the page in question.

Reply:
The store gets registered during import, so I can't figure out how can it be not initialized at this time. I'm not sure I share any concern about this change - perhaps I misunderstood something?